apis_configs/indexd_settings.py

```diff
@@ -28,6 +28,8 @@ def load_json(file_name):
 if dist:
     CONFIG["DIST"] = json.loads(dist)
 
+arborist = environ.get("ARBORIST", False)
```

Comment:
Maybe do something like:
```
arborist = (environ.get("ARBORIST", "False") == "True")
```
, otherwise if someone puts "False" or "True" in the manifest - this check will evaluate `True` below ...